content/applications/inventory_and_mrp/manufacturing/quality_control/quality_alerts.rst

```diff
@@ -0,0 +1,74 @@
+=====================
+Create quality alerts
+=====================
+
+:doc:`Configuring quality control points <quality_control_points>` is a great way to ensure that
+quality checks are performed at routine stages during specific operations. However, quality issues
+can often appear outside of these scheduled checks. Using *Odoo Quality*, users can create quality
+alerts for issues that are not detected by automated processes.
+
+Create quality alerts from the quality module
+=============================================
+
+In some situations, it is necessary to manually create quality alerts within the *Quality* module.
+
+.. example::
+   A helpdesk user who is notified of a product defect by a customer ticket can create an alert that
+   brings the issue to the attention of the relevant quality team.
+
+To create a new quality alert, start from the :menuselection:`Quality` module and select
+:menuselection:`Quality Control --> Quality Alerts --> Create`. The quality alert form can then be
+filled out as follows:
+
+- :guilabel:`Title`: choose a concise, yet descriptive title for the quality alert
+- :guilabel:`Product`: the product about which the quality alert is being created
+- :guilabel:`Product Variant`: the specific variant of the product that has the quality issue, if
+  applicable
+- :guilabel:`Lot`: the lot number assigned to the product
+- :guilabel:`Work Center`: the work center where the quality issue originated
+- :guilabel:`Picking`: the picking operation during which the quality issue originated
+- :guilabel:`Team`: the quality team that will be notified by the quality alert
+- :guilabel:`Responsible`: the individual responsible for managing the quality alert
+- :guilabel:`Tags`: classify the quality alert based on user-created tags
+- :guilabel:`Root Cause`: the cause of the quality issue, if known
+- :guilabel:`Priority`: assign a priority between one and three stars to ensure more
+  urgent issues are prioritized
+
+The tabs at the bottom of the form can be used to provide additional information to quality teams:
+
+- :guilabel:`Description`: provide additional details about the quality issue
+- :guilabel:`Corrective Actions`: the method for fixing affected products
+- :guilabel:`Preventive Actions`: procedures for preventing the issue from occurring in
+  the future
+- :guilabel:`Miscellaneous`: the product vendor (if applicable), the company that produces
+  the product, and the date assigned
+
+.. image:: quality_alerts/quality-alert-form.png
+   :align: center
+   :alt: An example of a completed quality alert form.
+
+Create quality alerts during the manufacturing process
+======================================================
```

Comment:
Nitpick:
Can choose a different word than "Create" just to add some variation in the headings 😉 
```suggestion
Add quality alerts during the manufacturing process
===================================================
```